gapis/api/vulkan/api/synchronization.api

```diff
@@ -48,6 +48,7 @@
 }
 
 @indirect("VkDevice")
+@sync_command
```

Comment:
The filtering of sync commands should only filter them inside the submit command (i.e. only filter them if they are sub-commands). While we may provide functionality to filter the host coordination group more narrowly in the future, for now, expanding that group puts the user into the debugging territory and this feature is intended to focus on profiling only. So, I think we should only put the flag on sync commands that are of the form `vkCmd*` and in the filter only filter them if they are sub-commands.

Reply:
I think the right way to do this is still label all sync commands as sync commands, regardless of whether they are host side or device side, and then change the filter condition to "if(isSync() && isSubcommand())".

Reply:
Sounds good.

Reply:
done